test_win.cmd

```diff
@@ -1,38 +1,56 @@
-@CLS
-
 @ECHO OFF
-@ECHO ** Run the NUnit tests from the Windows command line **
+
+@ECHO/
+@ECHO ***************************************
+@ECHO ** Run the NUnit tests for GEDKeeper **
+@ECHO ***************************************
 @ECHO/
```

Comment:
I really do not like keywords in upper case or with a capital first letter. Straight the eye hurts... Please fix it.

Reply:
Fixed.